app/views/layouts/_header.html.erb

```diff
@@ -21,16 +21,23 @@
             class: 'form-control select2') %>
         </div>
         <div class="navbar-form navbar-left">
+          <%= link_to 'Get our newsletter', "http://eepurl.com/bt5QCz", class: "btn btn-default" %>
```

Comment:
Let's use the environment variable rather than the URL.  Actually, let's set an instance variable `@mailchimp_url` that fetches the env variable and is passed along to the partial.